docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/netskope-source.md

```diff
@@ -46,14 +46,26 @@ Hover your mouse over the status icon to view a tooltip with details on the dete
 
 ## Getting a token from Netskope Portal
 
-Netskope REST APIs use an auth token to make authorized calls to the
+### Netskope REST API v2 
+
+Netskope REST APIv2 provides an easy way to extend the Netskope platform to build to use-cases specific to your organization. Endpoints cover key areas such as Events, Alerts, Reports, Clients and more. 
+
+To obtain a Netskope REST API v2 auth token, do the following:
+
+1. Log in to Netskope as the Tenant Admin.
+1. Go to the API portion of the Netskope, **Settings** > **Tools** > **Rest API v2**.
+1. Click "New Token", provide the token name and expiration duration, then add the following endpoints with READ privilege:
```

Comment:
1. Log in to Netskope as the Tenant Admin.
1. Go to the API portion of the Netskope, **Settings** > **Tools** > **Rest API v2**.
1. Click "New Token", provide the token name and expiration duration, then add the following endpoints with READ privilege: `/api/v2/events/dataexport/events/alert`, `/api/v2/events/dataexport/events/page`, `/api/v2/events/dataexport/events/infrastructure`, `/api/v2/events/dataexport/events/application`, `/api/v2/events/dataexport/events/network`, `/api/v2/events/dataexport/events/alert/audit` and `/api/v2/events/data/alert`